ledger/src/dust.rs

```diff
@@ -1258,9 +1257,9 @@ tag_enforcement_test!(DustWalletUtxoState);
 
 #[derive(Debug)]
 pub enum DustSpendError {
-    BackingNightNotFound(QualifiedDustOutput),
+    BackingNightNotFound(Box<QualifiedDustOutput>),
```

Comment:
breaking change. I'm assuming this is the large enum lint.

Reply:
that's correct, I'll add a note in the commit description to mention this is a breaking change.